dicoogle_web_api.yaml

```diff
@@ -1,1313 +1,1319 @@
-swagger: '2.0'
+openapi: 3.0.0
 info:
-  description: >-
-    Documentation of the server API to the web services provided by Dicoogle,
-    the open-source PACS archive.
+  description:
+    <p>Specification of the Dicoogle's PACS archive public API. This page describes all services available
+    by default to the common user of the Dicoogle open-source PACS archive.
+    In the given examples, the Demo website is used to try out the services. They may also be tried in
+    your local deploy of Dicoogle through the base path <a href="http://localhost:8080">http://localhost:8080</a>,
+    or another base path previously set. More information about Dicoogle configuration available in
+    <a href="https://bioinformatics-ua.github.io/dicoogle-learning-pack/">Dicoogle Learning Pack</a>.</p>
+
+    <p>Finally, the Dicoogle Team encourage you to try out the official Javascript client API,
+    dicoogle-client-js, available in <a href="https://github.com/bioinformatics-ua/dicoogle-client-js">GitHub</a>
+    and documented in detail in <a href="https://bioinformatics-ua.github.io/dicoogle-client-js/">GitHub Pages</a>.</p>
+
+    <p>Useful external links:</p>
+    <p>- <a href="http://www.dicoogle.com">Dicoogle Website</a> </p>
+    <p>- <a href="https://github.com/bioinformatics-ua/dicoogle">Dicoogle GitHub</a> </p>
+    <p>- <a href="https://bioinformatics-ua.github.io/dicoogle-learning-pack/">Dicoogle Learning Pack</a> </p>
+    <p>- <a href="https://bioinformatics-ua.github.io/dicoogle-api/javadoc/">Dicoogle Javadoc</a> </p>
+    <p>- <a href="https://github.com/bioinformatics-ua/dicoogle-client-js">Dicoogle Javascript Client</a> </p>
+    <br>
   version: 3.0.0
   title: Dicoogle
   contact:
-    name: Dicoogle
-    url: 'http://dicoogle.com/about/'
+    name: Support
+    url: http://dicoogle.com/about/
+    email: carlos.costa@ua.pt
```

Comment:
Has this email been agreed with? The "contact us" form is probably enough and leaves the team's addresses a bit more protected from spam.

Reply:
Thanks for pointing it out! I added it as it is disclosed in the Dicoogle's official website in the About section. However, as it was not discussed this time, I chose to remove the email.